src/VirtualClient/VirtualClient.Actions/HPLinpack/HPLinpackExecutor.cs

```diff
@@ -0,0 +1,408 @@
+                    }
+                }
+            }
+        }
+
+        private void ThrowIfPlatformIsNotSupported()
+        {
+            if (this.Platform != PlatformID.Unix)
+            {
+                throw new WorkloadException(
+                    $"The HPL workload is currently only supported on the following platform/architectures: " +
+                    $"'{PlatformSpecifics.LinuxX64}', '{PlatformSpecifics.LinuxArm64}'.",
+                    ErrorReason.PlatformNotSupported);
+            }
+        }
+
+        private async Task ConfigureMakeFileAsync(EventContext telemetryContext, CancellationToken cancellationToken)
+        {
+            string makeFilePath = this.PlatformSpecifics.Combine(this.HPLDirectory, "setup", "Make.Linux_GCC");
+            await this.ExecuteCommandAsync("mv", $"Make.UNKNOWN Make.Linux_GCC", this.PlatformSpecifics.Combine(this.HPLDirectory, $"setup"), false, telemetryContext, cancellationToken);
+
+            await this.fileSystem.File.ReplaceInFileAsync(
+                    makeFilePath, @"ARCH *= *[^\n]*", "ARCH = Linux_GCC", cancellationToken);
+
+            await this.fileSystem.File.ReplaceInFileAsync(
+                    makeFilePath, @"TOPdir *= *[^\n]*", $"TOPdir = {this.HPLDirectory}", cancellationToken);
+
+            if (this.CpuArchitecture == Architecture.Arm64)
+            {
+                await this.fileSystem.File.ReplaceInFileAsync(
+                    makeFilePath, @"LAdir *=", "LAdir = $(ARMPL_DIR)", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"LAinc *=", $"LAinc = $(ARMPL_INCLUDES)", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"LAlib *= *[^\n]*", "LAlib = /opt/arm/armpl_22.1_gcc-11.2/lib/libarmpl.a", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"LINKER *= *[^\n]*", "LINKER = mpifort", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"CCFLAGS *= *[^\n]*", "CCFLAGS = $(HPL_DEFS) -Ofast -march=armv8-a", cancellationToken);
+            }
+            else if (this.CpuArchitecture == Architecture.X64)
+            {
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"MPinc *=", $"MPinc =  -I/usr/lib/x86_64-linux-gnu/openmpi", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"MPlib *=", "MPlib =  /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi.so", cancellationToken);
+
+                await this.fileSystem.File.ReplaceInFileAsync(
+                        makeFilePath, @"LAinc *=", "LAinc = -I/usr/lib/x86_64-linux-gnu", cancellationToken);
+            }
+
+            await this.fileSystem.File.ReplaceInFileAsync(
```

Comment:
Is there a cflag replacement with the linux-x64? I don't see it

Reply:
Nope we don't default CCFLAG value generated in HPL.dat file while hpl build works well in linux-x64 case.